spec/views/coordinators/show.html.erb_spec.rb

```diff
@@ -0,0 +1,56 @@
+# spec/views/products/show.html.erb_spec.rb
+
+require 'rails_helper'
+require 'spec_helper'
+
+RSpec.describe 'coordinators/show.html.erb', type: :view do
+  describe 'populate page and links' do
+    before do
+      @coordinator = Coordinator.create(name: 'Joe Lally',
+                                        email: 'email@email.com')
+      @tutors = [Tutor.create(first_name: 'Bob', last_name: 'Smith'),
+                 Tutor.create(first_name: 'Tom', last_name: 'Brady')]
+      @students = [Student.create(first_name: 'Mike', last_name: 'White'),
+                   Student.create(first_name: 'Alex', last_name: 'Fallah')]
```

Comment:
As we add more required fields, it will become far too cumbersome to use `Student.create` and `Tutor.create`, and we will 100% want to pivot to using FactoryGirl.